.github/workflows/smoke-test.yml

```diff
@@ -0,0 +1,33 @@
+# This workflow will run daily at 06:45.
+# It will install Python dependencies and run tests with a variety of Python versions.
+
+name: Unit test smoke test
+
+on:
+  schedule:
+    - cron: 45 6 * * *
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        python-version: ['3.8', '3.9', '3.10']
+
+    steps:
+    - uses: actions/checkout@v3
+    - name: Set up Python {% raw %}${{ matrix.python-version }}{% endraw %}
+      uses: actions/setup-python@v4
+      with:
+        python-version: {% raw %}${{ matrix.python-version }}{% endraw %}
```

Comment:
I just created this PR in the python project template to fix this: https://github.com/lincc-frameworks/python-project-template/pull/115/files

But basically, if you remove the `{% raw %}` and `{% endraw %}` tags on line 20 and 23, then this should become valid YAML, and the test should pass.